src/client/components/MusicReserve/TrackTable.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View } from '@instructure/ui-view';
+import { CondensedButton } from '@instructure/ui-buttons';
+import { Table } from '@instructure/ui-table';
+
+export const TrackTable = ({ album, handleClick, goBack }) => {
+  TrackTable.propTypes = {
+    album: PropTypes.object,
+    handleClick: PropTypes.func,
+    goBack: PropTypes.func,
+  };
+
+  return (
+    <View>
+      <CondensedButton onClick={goBack} display="block" margin="medium">
+        {'< Back'}
+      </CondensedButton>
```

Comment:
How would it look if we used a proper [Button](https://instructure.design/#Button) and passed in [arrow-open-start](https://instructure.design/#iconography) to the renderIcon prop? Might look a bit nicer and easier to tap on mobile

```html
<Button renderIcon={IconArrowOpenStartSolid}>Back</Button>
```

<img width="970" alt="Screen Shot 2020-08-07 at 12 45 43 PM" src="https://user-images.githubusercontent.com/12589878/89682646-e9b23900-d8ab-11ea-85e5-afe1d2b761bd.png">

Reply:
+1 above, but also here is the breadcrumbs component to show to the user the navigation path to got to the content https://instructure.design/#Breadcrumb